service/service.go

```diff
@@ -123,7 +131,11 @@ func New(name string) *Service {
 		Router:     mux.NewRouter(),
 		ServeMux:   http.NewServeMux(),
 	}
+
+	s.Status.IsHealthy = true
+	s.Status.IsReady = false
```

Comment:
Service is never ready? 🤔

Reply:
It depends on your k8s definition whether you use the readiness probe or not.

I will change it to *true*, because in many cases running = healthy = ready.

Reply:
Ah, so the service should be in charge of setting this to `true`. Makes sense.

Reply:
Yes. However. Your comment helped me to realize that setting it to true will work for most of our lightweight micro-services.